src/main/java/com/mykaarma/urlshortener/exception/ShortUrlException.java

```diff
@@ -0,0 +1,53 @@
+package com.mykaarma.urlshortener.exception;
+
+
+import com.mykaarma.urlshortener.dto.UrlErrorDTO;
+import com.mykaarma.urlshortener.enums.UrlErrorCodes;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+@Getter
+@Setter
+public class ShortUrlException extends Exception {
```

Comment:
@AadiS11 Does this have to extend Exception? Or can we make it extend RuntimeException?